library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -179,6 +179,11 @@
     private ScrollListener mScrollListener;
     private AddEventClickListener mAddEventClickListener;
 
+    //Scroll by amount of days desired functionnality
+    private boolean mIsCustomScrollableAmountsOfDaysEnabled = false;
```

Comment:
Should have a better name, `mIsScrollByAmountOfVisibleDaysEnabled` for example

Reply:
Ok